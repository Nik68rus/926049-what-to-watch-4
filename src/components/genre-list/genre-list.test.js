import React from 'react';
import renderer from 'react-test-renderer';
import GenreList from './genre-list';

const genres = [`drama`, `horror`, `sci-fi`];
const activeGenre = `drama`;
const onGenreClick = jest.fn();

it(`GenreList renders correctly`, () => {
  const tree = renderer.create(<GenreList genres={genres} activeGenre={activeGenre} onGenreClick={onGenreClick} />).toJSON();
  expect(tree).toMatchSnapshot();
});
