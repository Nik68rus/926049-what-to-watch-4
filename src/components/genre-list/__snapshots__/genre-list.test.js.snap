// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList renders correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      drama
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      horror
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      sci-fi
    </a>
  </li>
</ul>
`;
